project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject kraken-works/lein-template "0.5.3-SNAPSHOT"
+(defproject kraken-works/lein-template "0.6.1-SNAPSHOT"
```

Comment:
Not a big deal _at all_, but typically PRs should leave the version alone. `lein release` handles all that. With this change, `lein release` would create and push 0.6.1, which isn't next.

Reply:
@tie-rack Sorry, I do understand this but I've done releases manually for too long. I will remove these commits.

Reply:
@tie-rack If the next release *wouldn't* be `0.5.3` (just hypothetically), then the PR should bump the `-SNAPSHOT` version though, right?

Reply:
@gws If the next one should be `0.6.0`, then we'll run `lein release :minor`.